options/locale/locale_en-US.ini

```diff
@@ -2284,6 +2284,7 @@ settings.tags.protection.none = There are no protected tags.
 settings.tags.protection.pattern.description = You can use a single name or a glob pattern or regular expression to match multiple tags. Read more in the <a target="_blank" rel="noopener" href="https://docs.gitea.io/en-us/protected-tags/">protected tags guide</a>.
 settings.bot_token = Bot Token
 settings.chat_id = Chat ID
+settings.thread_id = Thread ID
```

Comment:
We backport a change that adds translations?
Interesting…

Reply:
I am neutral for it and I think it doesn't cause any real problem.

Since this been backported without conflict and gets approvals, I think it's good to merge.